components/dashboard/src/service/public-api.ts

```diff
@@ -55,6 +57,8 @@ export const configurationClient = createServiceClient(ConfigurationService);
 
 export const authProviderClient = createServiceClient(AuthProviderService, new JsonRpcAuthProviderClient());
 
+export const scmClient = createServiceClient(SCMService, new JsonRpcScmClient());
```

Comment:
Bump version and add service (see last time we have refresh problem since not added it) in setup.tsx

Reply:
Oh, I also forgot to add known classes.